spec/stash_core_api/commits_spec.rb

```diff
@@ -59,4 +59,36 @@
       end
     end
   end
+
+  describe '#commit_changes' do
+    it 'should limit the results when the option is passed' do
+      endpoint = "/#{base_path(client)}/commits/083f66ba3c6/changes?limit=1"
+      stub_get(client, endpoint).to_return(
+        body: fixture('commit_changes.json'),
+        headers: { 'Content-Type' => 'application/json' }
+      )
+      changes = client.commit_changes('083f66ba3c6', limit: 1)
+      expect(changes['size']).to eq(1)
```

Comment:
If this test endpoint always returns 1 change, then is this really testing anything?

Reply:
+1

Reply:
true, I guess it would be more fruitful to just test that it's generating the right URL, which my intention here.